content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -0,0 +1,118 @@
+
+* [Connecting to Kubeflow Pipelines on Google Cloud using the SDK](/docs/distributions/gke/pipelines/authentication-sdk/)
+* [Kubeflow Pipelines on AWS](https://www.kubeflow.org/docs/distributions/aws/pipeline/#authenticate-kubeflow-pipeline-using-sdk-inside-cluster)
+* [Authentication using OIDC in Azure](/docs/distributions/azure/authentication-oidc/)
+* [Pipelines on IBM Cloud Kubernetes Service (IKS)](/docs/distributions/ibm/pipelines/)
+
+If the endpoint is accessible without authentication / authorization, it's easier to connect. Here are examples
+for [Kubeflow Pipelines standalone](https://www.kubeflow.org/docs/components/pipelines/installation/standalone-deployment/).
+
+### Connect to API from outside your cluster
+
+Kubeflow Pipelines standalone deploys a Kubernetes service named `ml-pipeline-ui` in your Kubernetes cluster without extra authentication.
+
+You can use [kubectl port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) to port forward the Kubernetes service locally to your laptop outside of the cluster:
+
+```bash
+# Change the namespace if you deployed Kubeflow Pipelines in a different
+# namespace.
+$ kubectl port-forward svc/ml-pipeline-ui 3000:80 --namespace kubeflow
+```
+
+You can verify port forwarding is working properly by visiting <http://localhost:3000> in your browser. Kubeflow Pipelines UI should show up.
+
+Then, you can create an SDK client in python by:
+
+```python
+import kfp
+client = kfp.Client(host='http://localhost:3000')
+print(client.list_experiments())
+```
+
+Note, for Kubeflow Pipelines in multi-user mode, you cannot access the API using kubectl port-forward
+because it requires authentication. Refer to distribution specific documentation as recommended above.
+
+### Connect to API from the same cluster
+
+Note, this is not supported right now for multi-user Kubeflow Pipelines, refer to [Multi-User Isolation for Pipelines -- Current Limitations](/docs/components/pipelines/multi-user/#current-limitations).
+
+As mentioned above, the Kubeflow Pipelines API Kubernetes service is `ml-pipeline-ui`.
+
+Using [Kubernetes standard mechanisms to discover the service](https://kubernetes.io/docs/concepts/services-networking/service/#discovering-services), we can access it from a Pod in the same namespace by DNS name:
+
+```python
+import kfp
+client = kfp.Client(host='http://ml-pipeline-ui:80')
+print(client.list_experiments())
+```
+
+Or by environment variables:
+
+```python
+import kfp
+import os
+host = os.getenv('ML_PIPELINE_UI_SERVICE_HOST')
+port = os.getenv('ML_PIPELINE_UI_SERVICE_PORT')
+client = kfp.Client(host=f'http://{host}:{port}')
+print(client.list_experiments())
+```
+
+When not in the same namespace, we can only access by DNS name with namespace:
```

Comment:
Suggested revision:

When accessing Kubeflow Pipelines from a Pod in a different namespace, you must specify the service name and the namespace: